movici_simulation_core/utils/attribute_loader.py

```diff
@@ -0,0 +1,67 @@
+"""Dynamic attribute loader from JSON configuration."""
+
+import json
+from pathlib import Path
+from typing import Any, Dict, List
+
+from movici_simulation_core import AttributeSpec, DataType
+
+
+def load_attributes(json_path: Path = None) -> List[AttributeSpec]:
+    """
+    Load attributes from JSON file and create AttributeSpec objects.
+
+    Args:
+        json_path: Path to attributes.json. Defaults to BASE_DIR/attributes.json
+
+    Returns:
+        List of AttributeSpec objects ready for register_attributes()
+    """
+    if json_path is None:
+        # Default to attributes.json in project base directory
+        json_path = Path(__file__).parent.parent.parent / "attributes.json"
```

Comment:
There is no guarantee that that is where the project directory is. That is fully dependent on the installation path.

I thiink maybe for now we should make the path a required argument, so that we don't make any wrong assumptions about a default path